### PR TITLE
[RHPAM-3784] - [GSS] (7.10.x) Prometheus exporter 7.10.1 : kie_server…

### PR DESCRIPTION
**Pull request by spolti**
…_container_started_total gives wrong count

Signed-off-by: spolti <fspolti@redhat.com>

**Thank you for submitting this pull request**

**JIRA**: https://issues.redhat.com/browse/RHPAM-3784

### CODE REVIEW

**Comment by spolti** [comment]
needs cherry-pick to 7.12.1 release branch.

File: kie-server-parent/kie-server-tests/kie-server-integ-tests-all/src/test/java/org/kie/server/integrationtests/prometheus/PrometheusIntegrationTest.java [hunk 1]
```diff
@@ -117,8 +117,11 @@ public void testKieServerStartAndContainersMetrics() {
         assertThat(getMetrics()).contains(
                 "kie_server_start_time{name=\"" + serverName + "\",",
                 "kie_server_deployments_active_total{deployment_id=\"prometheus\",} 1.0",
-                "kie_server_container_started_total{container_id=\"prometheus\",} 1.0",
-                "kie_server_container_running_total{container_id=\"prometheus\",} 1.0"
+                "kie_server_container_running{container_id=\"prometheus\",} 1.0",
+                "kie_server_container_started{container_id=\"prometheus\",} 1.0",
+                "kie_server_container_started_total 2.0",
+                "kie_server_container_running_total 2",
+                "kie_server_container_started_since_boot 2"
```
Comment (sutaakar) [inline kie-server-parent/kie-server-tests/kie-server-integ-tests-all/src/test/java/org/kie/server/integrationtests/prometheus/PrometheusIntegrationTest.java:124]: @spolti Can you please remove this line? Or remove value? This metric is affected by other integration tests too.

**Review by sutaakar — APPROVED** [review]

File: kie-server-parent/kie-server-tests/kie-server-integ-tests-all/src/test/java/org/kie/server/integrationtests/prometheus/PrometheusIntegrationTest.java (hunk 1 above)
Comment (spolti) [reply kie-server-parent/kie-server-tests/kie-server-integ-tests-all/src/test/java/org/kie/server/integrationtests/prometheus/PrometheusIntegrationTest.java:124]: these 4 lines?

**Comment by sutaakar** [comment]
Jenkins retest this please

**Review by cristianonicolai — APPROVED** [review]

**Comment by spolti** [comment]
@cristianonicolai @mareknovotny can this be merged?

**Pull request merged**